tools/karpenter-convert/pkg/convert/convert.go

```diff
@@ -161,20 +190,30 @@ func processNodeTemplate(resource runtime.Object) runtime.Object {
 	}
 
 	nodeclass := nodeclassutil.New(nodetemplate)
+	nodeclass.ObjectMeta.CreationTimestamp = metav1.Time{}
```

Comment:
```suggestion
	nodeclass.CreationTimestamp = metav1.Time{}
```

nit: can cut the `ObjectMeta` portion